crypto/dilithium/pqcrystals_dilithium_ref_common/api.h

```diff
@@ -25,53 +25,27 @@ int pqcrystals_dilithium2_ref_keypair(uint8_t *pk, uint8_t *sk);
 
 int pqcrystals_dilithium2_ref_signature(uint8_t *sig, size_t *siglen,
                                         const uint8_t *m, size_t mlen,
+                                        const uint8_t *ctx, size_t ctxlen,
                                         const uint8_t *sk);
 
 int pqcrystals_dilithium2_ref(uint8_t *sm, size_t *smlen,
                               const uint8_t *m, size_t mlen,
+                              const uint8_t *ctx, size_t ctxlen,
                               const uint8_t *sk);
 
 int pqcrystals_dilithium2_ref_verify(const uint8_t *sig, size_t siglen,
                                      const uint8_t *m, size_t mlen,
+                                     const uint8_t *ctx, size_t ctxlen,
                                      const uint8_t *pk);
 
 int pqcrystals_dilithium2_ref_open(uint8_t *m, size_t *mlen,
                                    const uint8_t *sm, size_t smlen,
+                                   const uint8_t *ctx, size_t ctxlen,
                                    const uint8_t *pk);
 
-#define pqcrystals_dilithium2aes_ref_PUBLICKEYBYTES pqcrystals_dilithium2_ref_PUBLICKEYBYTES
-#define pqcrystals_dilithium2aes_ref_SECRETKEYBYTES pqcrystals_dilithium2_ref_SECRETKEYBYTES
-#define pqcrystals_dilithium2aes_ref_BYTES pqcrystals_dilithium2_ref_BYTES
-
-#ifdef BORINGSSL_PREFIX
```

Comment:
I don't see the post-merge equivalent of this, is there more work to preserve the prefix builds?

Reply:
Nope, as called out in the PR header, we are removing AES based modes (these were offered by the authors of dilithium for performance, but NIST didn't want any of those modes).